vmdk_plugin/vmdkops/esx_vmdkcmd.go

```diff
@@ -99,11 +99,11 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 			errno = err.(syscall.Errno)
 			msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
 			if retryCount >= maxRetryCount {
-				if errno == syscall.ECONNRESET {
+				if errno == syscall.ECONNRESET || errno == syscall.ETIMEDOUT {
 					msg += " Hit communication issue with ESX (vmci or ESX service)\n"
```

Comment:
not a part of this fix but since you are touching the code here.., I think the message should be changed to simply " Failed to communicated with ESX. For details please refer to FAQ..."